### DISCUSSION TITLE
Email from Customers

### DISCUSSION THREAD

**Discussion by anmol1905** [vendurehq/vendure#1657] [Q&A]
Hello team,

In my project, there will be an option of a custom order, user can write what type of product they want in a form with the contact details and send it to us as email. I noticed that there are options for sending emails to the customers but is there any option in vendure from which users can send email?

Thanks a lot!

**Comment by michaelbromley** [comment]
Hi!
Sure, you could implement this using the exact same infrastructure as is currently used to send transactional emails to customers. Namely, you create a plugin which defines a custom event representing a customer filling out the form, so the even would contain all the form details. As part of the "submit" mutation you would use the [EventBus](https://www.vendure.io/docs/typescript-api/events/event-bus/) to publish that event.

Then you define a custom [EmailEventHandler](https://www.vendure.io/docs/typescript-api/email-plugin/email-event-handler/#emaileventhandler) which listens for that event and sends an email to your address containing the submitted data.
